parsl/channels/__init__.py

```diff
@@ -1,7 +1,4 @@
 from parsl.channels.base import Channel
 from parsl.channels.local.local import LocalChannel
-from parsl.channels.oauth_ssh.oauth_ssh import OAuthSSHChannel
-from parsl.channels.ssh.ssh import SSHChannel
-from parsl.channels.ssh_il.ssh_il import SSHInteractiveLoginChannel
 
-__all__ = ['Channel', 'SSHChannel', 'LocalChannel', 'SSHInteractiveLoginChannel', 'OAuthSSHChannel']
+__all__ = ['Channel', 'LocalChannel']
```

Comment:
Heh; a LocalChannel is a "fake" channel, eh?  :innocent:

Reply:
`LocalChannel` gets used both as a default (which can be edited by this PR) and as an explicit fake-default by a bunch of people (who copied it from examples) which this PR cannot edit.

That explicit fake-default path will have to be changed by users eventually, but it's more likely to affect users who don't care about channels - so I was avoiding that in this PR and leaving that inevitable pain till later...

The main point of this PR is to flush out the people who *do* care about (non-local) channels.